lib/main.dart

```diff
@@ -67,7 +67,7 @@ class _MyHomePageState extends State<MyHomePage> with TickerProviderStateMixin {
     try {
       currentLocation = await location.getLocation;
     } on PlatformException {
-      location = null;
+      print(MissingPluginException);
```

Comment:
I don't think this will work, it looks like it's printing a type instead of an exception instance.

It should look like this:

```
} on PlatformException catch(e) {
  print(e);
}
```